### DISCUSSION TITLE
Documentation on EFS usage is outdated

### DISCUSSION THREAD

**Discussion by bolah** [nextflow-io/nextflow#2825] [Q&A] [answered]
We were trying to follow the information available on github(code examples) and also the blog posts like https://www.nextflow.io/blog/2016/deploy-in-the-cloud-at-snap-of-a-finger.html

Would it be possible to update the documentation on how to use EFS with Nextflow? Since _cloud_ configuration is outdated, it is not possible to use cloud.sharedStorageId. Should we use aws.sharedStorageId instead?

**Comment by pditommaso** [comment, marked as answer]
Hello, support for AWS EFS comes with a separate plugin for Nextflow. You can find the documentation in the [corresponding repo](https://github.com/seqeralabs/xpack-amzn).
